tools/build-tools/src/repoPolicyCheck/handlers/assertShortCode.ts

```diff
@@ -0,0 +1,112 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    Node,
+    NumericLiteral,
+    Project,
+    SourceFile,
+    StringLiteral,
+    SyntaxKind,
+    TemplateLiteral,
+} from "ts-morph";
+import {Handler} from "../common";
+
+const shortCodes = new Map<number, Node>();
+const newAssetFiles = new Set<SourceFile>();
+let maxShortCode = -1;
+
+function getCallsiteString(msg: Node){
+    return `${msg.getSourceFile().getFilePath()}@${msg.getStartLineNumber()}`
+}
+
+function getAssertMessageParams(sourceFile: SourceFile): (StringLiteral | NumericLiteral | TemplateLiteral)[]{
+    const calls = sourceFile.getDescendantsOfKind(SyntaxKind.CallExpression);
+    const messageArgs:(StringLiteral | NumericLiteral | TemplateLiteral)[] = []
+    for(const call of calls){
+        if(call.getExpression().getText() === "assert"){
```

Comment:
I assume that will catch any function call to any "assert" function. I think it's Ok, but might be not expected by engineers

Reply:
it will. we block node types so there shouldn't be any other asserts in our code base. i'll see if i can find a place somewhere in a readme to add a blurb about it.